README.adoc

```diff
@@ -96,9 +96,7 @@ The first stage copies the [hotspot=copyJar]`guide-spring-boot-0.1.0.jar` Spring
 and then uses the Open Liberty [hotspot=springBootUtility]`springBootUtility` command to thin the application. 
 For more information about the `springBootUtility` command, see the https://www.ibm.com/support/knowledgecenter/en/SSEQTP_liberty/com.ibm.websphere.wlp.doc/ae/rwlp_command_sbutility.html[springBootUtility documentation^].
 
-The second stage begins with the [hotspot=OLimage2 file=0]`Open Liberty Docker image`. Then, the Dockerfile copies 
-the Spring Boot dependent library JAR files that are at [hotspot=libcache file=0]`lib.index.cache` and
-the [hotspot=thinjar file=0]`thin-guide-spring-boot-0.1.0.jar` file, both of which were generated in the first stage.
+The second stage begins with the [hotspot=OLimage2 file=0]`Open Liberty Docker image`. The Dockerfile copies the [hotspot=serverXml file=0]`server.xml` file from the `/opt/ol/wlp/templates` directory. Then, the Dockerfile copies the Spring Boot dependent library JAR files that are at [hotspot=libcache file=0]`lib.index.cache` directory and the [hotspot=thinjar file=0]`thin-guide-spring-boot-0.1.0.jar` file. The `lib.index.cache` directory and the `thin-guide-spring-boot-0.1.0.jar` file were both generated in the first stage.
```

Comment:
Can we explain a bit more about why this particular server.xml? Something along the lines of:

The Dockerfile copies the pre-configured Spring Boot [hotspot=serverXml file=0]`server.xml` file from the `/opt/ol/wlp/templates` directory, which enables Spring Boot and TLS support.

Reply:
updated.

@Charlotte-Holt are you OK for the change?

Reply:
@gkwan-ibm Yes, looks good. Thanks.